formula-android/src/main/java/com/instacart/formula/android/internal/ActivityStoreContextImpl.kt

```diff
@@ -104,7 +105,9 @@ internal class ActivityStoreContextImpl<Activity : FragmentActivity> : ActivityS
         attachEventRelay.accept(false)
     }
 
-    fun updateFragmentLifecycleState(contract: FragmentKey, newState: Lifecycle.State) {
+    fun updateFragmentLifecycleState(id: FragmentId, newState: Lifecycle.State) {
+        // TODO: should probably start using [id] instead of [contract] here.
```

Comment:
This is a bigger rethink as in other places we are listening for fragment events using only `FragmentKey`. It's a much smaller issue than this, so will perform these fixes in a follow-up.